### ISSUE TITLE
hibernation not working

### ISSUE THREAD

**Issue by xiaotianxt** [mohamedspicer/hackintosh-T480s#8]
Doesn't go to sleep mode when closing the laptop.

**Comment by m4salah** [comment]
I disable the hibernate, because of the long sleep the laptop doesn't wake up.

**Comment by m4salah** [comment]
when closing the laptop it's just close the screen but the laptop still running and consumes the battery if you solve the long sleep issue and make the hibernate working please pr the repo or type your solve here, Thanks.

**Comment by xiaotianxt** [comment]
> when closing the laptop it's just close the screen but the laptop still running and consumes the battery if you solve the long sleep issue and make the hibernate working please pr the repo or type your solve here, Thanks.

I'll try, but I'm afaid I may not be able to do that because I'm just a rookie😂.
However I do have a [repo](https://github.com/xiaotianxt/Hackintosh-T480s) that have no trouble with hibernation, but configured using Clover, I'm not sure how it was made.

**Comment by m4salah** [comment]
you can type this command in your clover machine and know what is your hibernate mode "pmset -g | grep hibernatemode"
and after you know what is the mode that working in clover machine you can apply it in the opencore with this command "sudo pmset -a hibernate mode value" and replace the value with the number that shows up earlier in clover. and test it if it works and show us the result.
worth mentioning that repo https://github.com/tylernguyen/x1c6-hackintosh he says that mode 25 works for him i didn't test it either.

**Comment by xiaotianxt** [comment]
Yeah I think 25 also works for me with clover, I'll try on opencore.

**Comment by xiaotianxt** [comment]
I really tried several hybernation mode but no one work. Please let me know if you fix this...

**Comment by m4salah** [comment]
i will keep this issue open until I find a solution or someone else, thank you for effort man

**Comment by xiaotianxt** [comment]
Hi, I just want you to know that I've figured out how to fix hibernation mode, I found a repo with good hibernation mode performance and are easy to use:
[opflep T480s_Hackintosh](https://github.com/opflep/T480s_Hackintosh)

**Comment by wangyv6** [comment]
i met this problem too.  screen becomes black when computer sleeps. but when i press delete button , it still responds( alert sound）

**Comment by xiaotianxt** [comment]
> i met this problem too. screen becomes black when computer sleeps. but when i press delete button , it still responds( alert sound）

兄弟看我上面的回复，那个EFI好使的

**Comment by wangyv6** [comment]
hi，
  谢谢， 我试过你发的efi了 ，但还是遇到了同样的问题（睡眠唤醒后黑屏，但是按delete键有提示音）。笔记本独显版本的，不知道是不是这个原因；另外 我是把4g模块改造成硬盘装mac的。（之前用clover一切正常...
  😂
发自我的手机

在 2021年2月5日，下午9:27，XiaoTian <notifications@github.com> 写道：

﻿

i met this problem too. screen becomes black when computer sleeps. but when i press delete button , it still responds( alert sound）

兄弟看我上面的回复，那个EFI好使的

**Comment by nmcc1212** [comment]
@xiaotianxt  can you please say what you changed so that hibernation will work

translation: 你能说出你改变了什么以便冬眠有效吗
